ProfileService.js

```diff
@@ -1,35 +1,26 @@
 /*!
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
-'use strict';
 
-import axios from 'axios';
+import {httpClient} from '@digitalbazaar/http-client';
 
 /**
  * @param {object} [config = {urls: {base: '/profiles'}}] - The config options.
  * @param {string} [config.baseURL] - Protocol, host & port used with Node.js
  *   such as https://example.com.
- * @param {object} [config.httpsAgent] - An optional
- *   node.js `https.Agent` instance to use when making requests.
  * @param {object} [config.urls = {}]
  * @param {string} [config.urls.base = 'FIXME']
  */
 export class ProfileService {
   constructor({
     baseURL,
-    httpsAgent,
     urls = {
       base: '/profiles',
       profileAgents: '/profile-agents'
     }
   } = {}) {
     this.config = {urls};
-    const headers = {Accept: 'application/ld+json, application/json'};
-    this._axios = axios.create({
-      baseURL,
-      headers,
-      httpsAgent,
```

Comment:
We need to continue to pass the `httpsAgent` into the `ProfileService`. Not doing so, hampers our usage in NodeJS vs. the browser.

Reply:
@gannan08 isn't this library a browser lib? My understanding is that an https agent is not required in browser code because the browser deals with all the stuff around self-signed certificates.

Reply:
It is not exclusively a browser lib. All bedrock-web* libraries can be thought of as client web service libraries for bedrock-* libraries. When a library primarily executes HTTP Requests, then the library will be isomorphic. If for any reason we are accessing Browser APIs (CHAPI, LocalStorage, IndexedDB, etc.), then it will be browser only. This is the general rule that I follow. Others may chime in.

Some examples: `bedrock-web-session` and `bedrock-session-http`, `edv-client` and `bedrock-edv-storage`, `webkms-client` and `bedrock-kms-http`, etc.

cc: @dlongley @mattcollier

Reply:
It was my understanding that `bedrock-web-foo` modules were browser code and were not intended to be isomorphic.  I thought the model was that `bedrock-web-foo` modules were for handling business logic and `bedrock-vue-foo` modules were for providing UI.

@gannan08 you mentioned this module and `bedrock-web-session`.  Are you aware of actual usage of these modules in backend code?

Reply:
Up to this point, `bedrock-web-foo` packages were intended to be browser code only -- not isomorphic. I don't think we've ever used one in node land yet. So I think @mattcollier's perspective reflects where things have been to date.

Reply:
@JSAssassin The team has spoken, continue with the removal of `httpsAgent`.